Startup.cs

```diff
@@ -4,6 +4,7 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using Microsoft.OpenApi.Models;
+using This_is_variant_music_api.Storage;
 
 namespace This_is_variant_music_api
```

Comment:
What did we decide on regarding casing convention for namespace?

Reply:
I think PascalCase is the norm for namespaces. Maybe something simpler, like `VariantMusicApi`?

Reply:
Yes, that's better 👌